backend/services/deviceconnect/api/http/management.go

```diff
@@ -346,15 +334,7 @@ func (h ManagementController) websocketWriter(
 		return err
 	}
 
-	pingPeriod := (pongWait * 9) / 10
-	ticker := time.NewTicker(pingPeriod)
-	defer ticker.Stop()
-	conn.SetPongHandler(func(string) error {
-		ticker.Reset(pingPeriod)
-		return conn.SetReadDeadline(time.Now().Add(pongWait))
-	})
 	conn.SetPingHandler(func(msg string) error {
-		ticker.Reset(pingPeriod)
 		err := conn.SetReadDeadline(time.Now().Add(pongWait))
```

Comment:
Same here. Could you remove this one too?
```suggestion
```

Reply:
There's another call on line 331:
https://github.com/mendersoftware/mender-server/blob/ceb2a79f4b41bc014f5f91094f44f682b1cbc986/backend/services/deviceconnect/api/http/management.go#L331-L335